packages/flutter/test/cupertino/checkbox_test.dart

```diff
@@ -466,4 +468,92 @@ void main() {
         ),
     );
   });
+
+  testWidgets('Checkbox fill color resolves in enabled/disabled states', (WidgetTester tester) async {
```

Comment:
Shall we add another test to verify that `fillColor` takes precedence over `activeColor` and `inactiveColor`?